app/Http/Controllers/PrintController.php

```diff
@@ -3,16 +3,51 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Storage;
 use Illuminate\Support\Facades\Validator;
 use App\User;
 use App\PrintAccount;
+use App\PrintJob;
 
 class PrintController extends Controller
 {
     public function index() {
         return view('print.app');
     }
 
+    public function print(Request $request) {
+        $validator = Validator::make($request->all(), [
+            'file_to_upload' => 'required|file|mimes:pdf|max:120000',
+            'number_of_copies' => 'required|integer|min:1'
+        ]);
+        $validator->validate();
+        
+        $print_account = Auth::user()->printAccount;
+        $two_sided = $request->has('two_sided');
+        $file = $request->file_to_upload;
+        $pages = exec("pdfinfo " . $file->getPathName() . " | grep '^Pages' | awk '{print $2}' 2>&1");
+
+        if($pages == "" || !is_numeric($pages) || $pages == 0){
+            Log::error("Cannot get number of pages for uploaded file!" . print_r($pages, true));
+            $validator->errors()->add('file_to_upload', __('internet.invalid_pdf'));
```

Comment:
Missing definition for `internet.invalid_pdf` both in Eng and Hun language translations.